l10n_it_fatturapa_in/wizard/wizard_import_fatturapa.py

```diff
@@ -1163,10 +1163,68 @@ def set_welfares_fund(
             return
 
         WelfareFundLineModel = self.env['welfare.fund.data.line']
+
+        wts = False
+        if any([welfareLine for welfareLine in Welfares
+                if welfareLine.TipoCassa == 'TC07']):
+            # Search the matching withholding tax
+            wts = self.env['withholding.tax'].search(
+                [('wt_types', '=', 'enasarco')])
+
+        processedLines = []
         for welfareLine in Welfares:
-            WalferLineVals = self._prepareWelfareLine(
-                invoice.id, welfareLine)
+            # Handle only TC07
+            if welfareLine.TipoCassa == 'TC07':
+                if not wts:
+                    raise UserError(_(
```

Comment:
Cercherei di evitare di bloccare di default l'importazione. Vedo 2 strade

- caricare dei dati di default per coprire i casi più comuni e che servano anche come esempio per eventuali altre configurazioni
 - invece di sollevare eccezione, salvare gli errori nelle inconsistenze

Reply:
@eLBati Qui la situazione è che 

> Vedi anche travis
> 
> ```
> 2019-09-24 09:11:42,000 6053 ERROR openerp_test odoo.addons.l10n_it_fatturapa_in.tests.test_import_fatturapa_xml: ` IntegrityError: duplicate key value violates unique constraint "account_account_code_company_uniq"
> 
> 2019-09-24 09:11:42,001 6053 ERROR openerp_test odoo.addons.l10n_it_fatturapa_in.tests.test_import_fatturapa_xml: ` DETAIL:  Key (code, company_id)=(whtaxpay, 1) already exists
> ```
> 
> Grazie

Ok! Fixato!

Reply:
> Cercherei di evitare di bloccare di default l'importazione. Vedo 2 strade
> 
> * caricare dei dati di default per coprire i casi più comuni e che servano anche come esempio per eventuali altre configurazioni
> * invece di sollevare eccezione, salvare gli errori nelle inconsistenze

@eLBati In questo contesto non possiamo avere dati di default, quindi andrei per salvare gli errori nelle consistenze

Reply:
@eLBati Ricontrollando il codice ho visto che il withholding non gestisce gli errori, viene usata la raise, magari per adesso lasciamo così e facciamo una PR per cambiare questo comportamento dove necessario?